### ISSUE TITLE
Scalar Metadata example

### ISSUE THREAD

**Issue by louisa-ba** [anvilco/spectaql#60]
scalar metadata does not produce a example

e.g.
```
{
  "SCALAR": {
    "DateTime": {
      "documentation": { 
        "example": "2016-10-07T01:08:03.405Z"
      }
    }
  }
```

produces

```
        <div id="definition-DateTime" class="definition panel" data-traverse-target="definition-DateTime">
            <div class="definition-tags">
              <a class="label" href="#tag-types">Types</a>
            </div>
            <h2 class="panel-title">
              <a name="/definitions/DateTime"></a>
              <p><code>DateTime</code></p>
            </h2>
            <div class="doc-row">
              <div class="doc-copy">
                <section class="json-schema-description">
                  <p>The <code>DateTime</code> scalar type represents a DateTime value as specified by
                    <a href="https://en.wikipedia.org/wiki/ISO_8601">iso8601</a>.</p>
                </section>
              </div>
              <div class="doc-examples">
                <section>
                  <h5>Example</h5>
                  <html>
                  <head></head>
                  <body><pre><code class="hljs language-gql"><span class="hljs-symbol">object</span>
</code></pre> </body>
                  </html>
                </section>
              </div>
            </div>
          </div>
```

**Comment by shawncarr** [comment]
Looking for how to document a custom scalar as well.

**Comment by newhouse** [comment]
@shawncarr @louisa-ba 

Thanks for using SpectaQL and thanks for writing in. Indeed, examples for default and custom scalars was not supported in SpectaQL, but [this PR](https://github.com/anvilco/spectaql/pull/79) adds it.

You were doing things correctly:
- Add the example to the `metadata.json` file
- OR get it into your Introspection Query results already
- OR use dynamic example generators to add examples dynamically during the build.

Stay tuned for this to be merged and released

**Comment by newhouse** [comment]
In `main` now, and published in `0.6.0`!
